common/lib/xmodule/xmodule/html_module.py

```diff
@@ -457,6 +457,7 @@ class CourseInfoFields:
 
 
 @XBlock.tag("detached")
+@XBlock.needs('replace_urls')
```

Comment:
Why do we need it for this XBlock? I've tried running without it, and static URLs were replaced correctly, without showing deprecation warnings.

Is there some edge case where this is used?

Reply:
This is required for the course updates.